Classes/Notifications/NotificationSubscriptionsController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  NotificationSubscriptionsController.swift
+//  Freetime
+//
+//  Created by Ryan Nystrom on 12/18/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+final class NotificationSubscriptionsController {
+
+    enum State {
```

Comment:
Where did you get this from? I love it 😀

Reply:
Haha, what do you mean?

Reply:
Using a `State` enum inside of a `Controller` :)